packages/mds-geography-service/repository/index.ts

```diff
@@ -240,6 +241,55 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
     }
   }
 
+  public isPublished = (geography: GeographyDomainModel) => {
+    return Boolean(geography.publish_date)
+  }
+
+  /**
+   * Deletes geographyMetadata by geography_id
+   * @param geography_id
+   * @returns geography_id
```

Comment:
Why return the input? If it's just a yes/no return, why not return true? Or we could return the geography object.

Not a big deal, just curious.

Reply:
Just a decade of habit. There've been so many guides on the "right way to REST", that I prefer the simplest. Like, for "delete()" having true/false return type can be desirable, but i dislike the sematics of that because "delete()" is a command, not a question like "isDeleted()". 
Also, when deleting something, I assume it's because the data was hated and unwanted, so if you _needed_ a copy of it you should be calling "get()" before you "delete()". 
Those are my 3 cents!